burnman/composite.py

```diff
@@ -72,6 +73,21 @@ def __init__(self, phases, fractions=None, fraction_type='molar'):
             self.molar_fractions = None
 
         self.set_averaging_scheme('VoigtReussHill')
+        self.name=name
+        
+    def __str__(self):
```

Comment:
This is unrelated to the PR. I assume you want it for the tests? We have ``to_string()`` in Material, Composite, etc.  which is not very pythonic but and this is very similar to ``debug_print`` (did you see this function?). The ``self.name`` should probably appear in ``to_string()`` and/or ``debug_print`` if you insist on keeping things this way.

Reply:
Yes, I should probably have put this in a separate PR. 
I think it might be worthwhile replacing `to_string()` with `__str__()` in all classes. That way, when anyone wants to see key information about an object, they can just type `print(object)`, rather than `print(object.to_string())`. What do you think?

Reply:
For the time being, I've just added self.name to to_string and debug_print.